packages/github/src/github.ts

```diff
@@ -14,8 +14,8 @@ export class GitHub extends Octokit {
     variables?: Variables
   ) => Promise<GraphQlQueryResponse>
 
-  constructor(token: string) {
-    super({auth: `token ${token}`})
+  constructor(token: string, opts: Omit<Octokit.Options, 'auth'> = {}) {
```

Comment:
Rather than omitting auth, can we just overload the constructor to something like:
```
constructor(token: string) { ... }
constructor(opts = {}) { ... }
```

That would also resolve #106

Reply:
True, but then how would we construct the authorization header for GraphQL requests?

Reply:
Good point